packages/zwave-js/src/lib/controller/Controller.ts

```diff
@@ -678,6 +692,62 @@ export class ZWaveController extends EventEmitter {
 		});
 	}
 
+	private async secureBootstrap(node: ZWaveNode): Promise<void> {
+		// If security has been set up and we are allowed to include the node securely, do it
+		if (
+			this.driver.securityManager &&
+			!this._includeNonSecure &&
+			node.supportsCC(CommandClasses.Security)
```

Comment:
The only problelm here is that when replacing a node this check always fails, how should I handle this? I need to fetch command classes before the secure bootstrap (like in add node request)